js/misc/paygManager.js

```diff
@@ -125,6 +125,14 @@ var PaygManager = new Lang.Class({
     get expiryTime() {
         return this._expiryTime;
     },
+
+    get isLocked() {
+        if (!this.enabled)
+            return false;
+
+        return this._expiryTime < (GLib.get_real_time() / 1e6)
```

Comment:
I assume there isn’t a `GLib.USEC_PER_SEC` constant available?

Reply:
Technically, this should be `<=`, not `<`.

Reply:
> I assume there isn’t a GLib.USEC_PER_SEC constant available?

Actually there is, will change that.

> Technically, this should be <=, not <.

Correct